src/guiapi/src/MItem_print.cpp

```diff
@@ -5,8 +5,8 @@
 /*****************************************************************************/
 //MI_NOZZLE
 MI_NOZZLE::MI_NOZZLE()
-    : WI_SPIN_U16_t(uint16_t(marlin_vars()->target_nozzle),
-        MenuVars::nozzle_range.data(), _(label), 0, true, false) {}
+    : WI_SPIN_U16_t(uint16_t(marlin_vars()->target_nozzle), "\177C",
```

Comment:
Declare a set of units and reuse them! I don't like the repeated strings here, do not rely on potential merging of duplicit strings